components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -21,34 +22,55 @@ function ProfileDetailCard({
   badges,
   totalBadgeCount,
   currentBadge,
-  nextGoal,
+  completedGoal,
+  completedDate,
 }: ProfileDetailCard) {
   return (
-    <div className="flex flex-col h-full gap-4 text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100">
-      <div className="flex flex-col items-center">
+    <div
+      className="grid grid-cols-4 h-full w-full text-sm border-2 bg-slate-50 text-slate-800 border-slate-800 hover:bg-violet-100"
+      style={{ height: "340px", objectFit: "contain" }}
+    >
+      <div className="flex ml-3 place-items-center col-span-4 mt-4">
         <img
-          className="items-center w-20 mt-4 border-3 rounded-full border-slate-800 "
+          className="col-span-1 w-20 my-3 border-3 rounded-full border-slate-800 "
           src={avatar}
         />
-        <p className="font-bold "> {name}</p>
+        <div className="col-span-3 items-center gap-1 mx-4">
+          <p className="font-bold mb-1"> {name}</p>
+
+          <JoinedDateComponent
+            user={name}
+            textSize={"small"}
+            createdAt={joinDate}
+          />
+          <BadgesDisplayedComponent
+            earnedBadges={badges}
+            textSize={"small"}
+            totalBadges={totalBadgeCount}
+          />
+        </div>
       </div>
-      <div className="flex flex-col items-start gap-4 mx-4 mb-4">
-        <JoinedDateComponent
-          user={name}
-          textSize={"small"}
-          createdAt={joinDate}
-        />
-        <BadgesDisplayedComponent
-          earnedBadges={badges}
-          textSize={"small"}
-          totalBadges={totalBadgeCount}
-        />
-        <p className="font-bold">Current Focus:</p>
+      <div className="ml-4 col-span-4 p-1 mb-4">
+        <p className="ml-2 font-bold">Current Focus:</p>
         <div className="flex items-center p-2 ">
-          <img src={currentBadge?.image} className="w-16 mr-4 rounded-full" />
-          <p className="underline "> {currentBadge?.title}</p>
-        </div>{" "}
-        <p className="underline">Next Goal: {nextGoal}</p>
+          <img
+            src={currentBadge?.image}
+            className="col-span-1 items-center w-16 mr-5 rounded-full"
+          />
+          <p className="col-span-3 underline"> {currentBadge?.title}</p>
+        </div>
+        <p className="ml-2 font-bold">Last Goal:</p>
+        <div className="flex items-center p-2">
+          <div className="col-span-1 mr-5">
+            <div className="col-span-1 bg-slate-800 w-16 h-16 border-black rounded-full flex items-center justify-center">
```

Comment:
want to take a stab at creating a circle using an svg? 

what if you have a parent container that has relative sizing i.e a <div className= 'w-1/6'> then put an svg inside the parent container and use one of the object-fit [properties](https://tailwindcss.com/docs/object-fit)

this might be helpful: https://medium.com/@masonk000/using-svgs-with-tailwind-css-84e9e3526b63